### PR TITLE
add: Next UI

### LINKED ISSUE #1762: [Add] NextUI within NextJs sub-category
### Site URL

https://nextui.org/

### Description of the above provided site

Beautiful, fast and modern React UI library.
NextUI provides a custom TailwindCSS plugin that allows you to customize the default themes or create your own.
NextUI components are built on top of [React Aria](https://react-spectrum.adobe.com/react-aria/) ensuring exceptional accessibility support as a top priority.

### Category

Frontend

### Sub Category

Next Js

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [x] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by DylanP97**
## Fixes Issue

Closes #1762

## Changes proposed

Adding NextUI link

## Screenshots

Untitled

### CODE REVIEW

**Comment by CBID2** [comment]
@DylanP97. Can you fill out the PR form please? It’ll help us gain a better understanding of your contribution.

**Comment by DylanP97** [comment]
> @DylanP97. Can you fill out the PR form please? It’ll help us gain a better understanding of your contribution.

Done let me know

File: database/frontend/next.json [hunk 1]
```diff
@@ -5,5 +5,12 @@
       "url": "https://nextjs.org/",
       "category": "frontend",
       "subcategory": "next-js"
+    },
+    {
+      "name": "NextUI",
+      "description": "Beautiful, fast and modern React UI library. NextUI provides a custom TailwindCSS plugin that allows you to customize the default themes or create your own. NextUI components are built on top of React Aria ensuring exceptional accessibility support as a top priority.",
```
Comment (CBID2) [inline database/frontend/next.json:11]:
```suggestion
      "description": "NextUI provides a custom TailwindCSS plugin that allows you to customize the default themes or create your own. Its components are built on top of React Aria ensuring exceptional accessibility support as a top priority.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @DylanP97! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Review by CBID2 — APPROVED** [review]
Well done

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by Anmol-Baranwal** [comment]
@CBID2 
Dont merge yet.

**Comment by Anmol-Baranwal** [comment]
@CBID2 @rupali-codes 
Next UI is already available in UI Generators category, should we add it here then. If not, then we can just recommend changing the resource name.

**Comment by CBID2** [comment]
> @CBID2 @rupali-codes 
> 
> Next UI is already available in UI Generators category, should we add it here then. If not, then we can just recommend changing the resource name.

Ooh good point @Anmol-Baranwal. @DylanP97, can you replace this one with a different source?

**Comment by DylanP97** [comment]
What about if I replace with Next Auth
https://next-auth.js.org/

**Comment by CBID2** [comment]
> What about if I replace with Next Auth
> 
> https://next-auth.js.org/

Is that source in there @Anmol-Baranwal?

**Comment by Anmol-Baranwal** [comment]
> > What about if I replace with Next Auth
> > https://next-auth.js.org/
> 
> Is that source in there @Anmol-Baranwal?

Yes!

image

@DylanP97 
You can just search it on LinksHub deployed link.

I'm giving you a suggestion, 
https://github.com/officialrajdeepsingh/awesome-nextjs

Don't start making issues for everything.😅

**Comment by Anmol-Baranwal** [comment]
update the issue description, title, pr description and title based on what you select. And please let me know so that I can check it first.

**Comment by DylanP97** [comment]
> Update the
> 
> * issue description,
> * issue title
> * pr description
> * pr title
> 
> based on what you select. And please let me know so that I can check it first.

Alright 
Sorry I didn't pay attention if it was in other categories
Let me come up with something unique

**Comment by CBID2** [comment]
@DylanP97, why did you close this pull request?

**Comment by Anmol-Baranwal** [comment]
@DylanP97 
We encourage making edit in same PR because, these closed ones are shown on your profile, and we certainly don't want that for you. 

Still, no worries and thanks for letting us know.

**Pull request closed without merging**